providers/shared/components/externalnetwork/id.ftl

```diff
@@ -84,8 +84,18 @@
                         "Names" : "Security",
                         "Type" : STRING_TYPE,
                         "Default" : "default"
+                    },
+                    {
+                        "Names" : "Alert",
+                        "Type" : STRING_TYPE,
```

Comment:
In light of https://github.com/hamlet-io/engine/pull/1514, requesting the following change for consistency

```suggestion
                        "Types" : STRING_TYPE,
```